static/_redirects

```diff
@@ -7,6 +7,12 @@
 # Customer Service
 /docs/customer-service/v1/*      https://docs-customer-service.netlify.app/docs/customer-service/v1/:splat         200
 
+# Rackspace SDDC Business Handbook
+/docs/sddc/sddc-business-handbook/*   https://docs.rackspace.com/docs/sddc/sddc-business-handbook/   200
+
+# Rackspace SDDC Enterprise Handbook
+/docs/sddc/sddc-enterprise-handbook/* https://docs.rackspace.com/docs/sddc/sddc-enterprise-handbook/   200
+
```

Comment:
Did you test the links and the UI experience before adding these redirects? Did they not work and is that why you had to add these?

I added a redirect on last week down below on lines 94 or 100 of this same file with:
```
# Private Cloud SDDC Docs
/docs/sddc/*    https://docs-private-cloud-sddc.netlify.app/docs/sddc/:splat    200
```

Reply:
When I click on card link it is showing 404 error. So, thought of updating redirect file.

Reply:
Now links are working.